bin/configs/python_aws_sigv4.yaml

```diff
@@ -0,0 +1,7 @@
+generatorName: python
+outputDir: samples/openapi3/client/petstore/python_aws_sigv4
```

Comment:
How about renaming the spec file something like signing_spec.yaml?
How about outputting the sample into the features directory?
How about changing the package name to something like signing_spec_api?
This sample is not related to petstore.